docs/generated/packages/angular/generators/component-cypress-spec.json

```diff
@@ -11,27 +11,32 @@
       "projectName": {
         "type": "string",
         "description": "The name of the project.",
-        "examples": ["ui-samples"]
+        "examples": ["ui-samples"],
+        "x-priority": "important"
       },
       "projectPath": {
         "type": "string",
         "description": "Path to the project.",
-        "examples": ["libs/ui-samples"]
+        "examples": ["libs/ui-samples"],
+        "x-priority": "important"
       },
       "componentName": {
         "type": "string",
         "description": "Class name of the component.",
-        "examples": ["AwesomeComponent"]
+        "examples": ["AwesomeComponent"],
+        "x-priority": "important"
       },
       "componentPath": {
         "type": "string",
         "description": "Relative path to the component file from the project root.",
-        "examples": ["awesome"]
+        "examples": ["awesome"],
+        "x-priority": "important"
```

Comment:
`required` properties are automatically going to be treated as `important`, so don't set these twice. Otherwise there will just be more maintenance overhead

Reply:
I've gone through and replaced some of them, but as long as it doesn't break functionality, i think this is less of a concern